insights/parsers/getconf_pagesize.py

```diff
@@ -0,0 +1,32 @@
+"""
+getconf PAGE_SIZE - command ``/usr/sbin/getconf PAGE_SIZE``
```

Comment:
Nitpick: Would you mind to change this title to "GetconfPageSize" correspondingly